src/components/MultiHopTrade/components/MultiHopTradeConfirm/utils/getChainShortName.ts

```diff
@@ -0,0 +1,38 @@
+import { KnownChainIds } from '@shapeshiftoss/types'
+import { assertUnreachable } from 'lib/utils'
+
+export const getChainShortName = (chainId: KnownChainIds) => {
```

Comment:
What's the rationale of going with this vs. the usual asset symbol? Is that something we want to generalize?

Reply:
it's for display purposes primarily, since we want a shorthand way to display assets on chains without confusing users:
- `Ethereum on Polygon` as `ETH.POLY` rather than `ETH.MATIC`
- `Ethereum on Optimism` as `ETH.ETH` rather than `ETH.OP`
- `Atom on Cosmos` as `ATOM.COSM` rather than `ATOM.ATOM`
- `Ethereum on Arbitrum Nova` as `ETH.ARB-Nova` rather than `ETH.ARB`